ecommerce/serializers.py

```diff
@@ -48,40 +48,35 @@ class ProductVersionSerializer(serializers.ModelSerializer):
     """ ProductVersion serializer for viewing/updating items in basket """
 
     type = serializers.SerializerMethodField()
-    course_runs = serializers.SerializerMethodField()
+    courses = serializers.SerializerMethodField()
     thumbnail_url = serializers.SerializerMethodField()
 
     def get_type(self, instance):
         """ Return the product version type """
         return instance.product.content_type.model
 
-    def get_course_runs(self, instance):
+    def get_courses(self, instance):
         """ Return the course runs in the product """
-        course_runs = []
-        if instance.product.content_type.model == "courserun":
-            course_runs.append(instance.product.content_object)
-        elif instance.product.content_type.model == "course":
-            course_runs.append(instance.product.content_object.first_unexpired_run)
-        elif instance.product.content_type.model == "program":
-            for course in Course.objects.filter(
+        model = instance.product.content_type.model
+        if model == "course":
+            courses = [instance.product.content_object]
+        elif model == "program":
```

Comment:
Take it or leave it: I think you can alternatively do this as `instance.product.content_type.model_class() is Course` and so forth

Reply:
:+1: